benches/compile_time.rs

```diff
@@ -44,7 +44,7 @@ pub fn bench_compile_time(c: &mut Criterion) {
                 b.iter(|| {
                     let native_context = NativeContext::new();
                     let module = native_context.compile(black_box(program)).unwrap();
-                    let object = module_to_object(module.module())
+                    let object = module_to_object(module.module(), OptLevel::Aggressive)
```

Comment:
Is this one set to `Aggressive` to test the worst case? Maybe also try best case with `None` or `Default`?

Reply:
Wouldn't aggressive be the best case?

Reply:
In compilation time? I expect aggressive optimizations to take longer.

Reply:
Oh right, I didn't notice what was being benchmarked. Thanks.